spec.bs

```diff
@@ -1704,7 +1704,8 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   To <dfn>canonicalize a protocol</dfn> given a string |value|:
 
   1. If |value| is the empty string, return |value|.
-  1. Let |dummyURL| be a new [=URL record=].
+  1. Let |dummyInput| be "http://dummy.test".
+  1. Let |dummyURL| be the result of running the [=basic URL parser=] on |dummyInput|.
   1. Let |parseResult| be the result of running the [=basic URL parser=] given |value| followed by "`://dummy.test`", with |dummyURL| as <i>[=basic URL parser/url=]</i>.
```

Comment:
Since we're not using state override I think we don't have to pass dummyURL either. This seems like an incorrect invocation of the basic URL parser at the moment.